build.gradle

```diff
@@ -26,10 +26,13 @@ minecraft {
 }
 
 dependencies {
-	//to change the versions see the gradle.properties file
+	// To change the versions see the gradle.properties file
 	minecraft "com.mojang:minecraft:${project.minecraft_version}"
 	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
 	modCompile "net.fabricmc:fabric-loader:${project.loader_version}"
+
```

Comment:
It doesn't look like the changes in this file are necessary for your commit at all. Where do you need the javax annotations?

Reply:
idk, maybe there is some bug. These annotations are required at compilation and if they are not there will crash.